### ISSUE TITLE
Not an issue but a contribution maybe?

### ISSUE THREAD

**Issue by AScustomWorks** [Aepelzen/AepelzensModules#15]
Hi there!, I made some small tweaks to the Gate seq panel design for my own use (font, bg and text colors). Maybe you would want to use it as a theme for all the modules? I can give you the info for the color values, font and font sizes if you like it:
![aepelzensgateseq](https://user-images.githubusercontent.com/34798768/37923821-5be11794-30ed-11e8-8bf3-c9d16d88172d.jpg)

**Comment by Aepelzen** [comment]
Hi,
thanks for the offer. Those fonts definitely look better than mine. I'm not sure if I ever get to redesigning all panels but if I do this would be a good starting point. So please leave the infos here.

**Comment by AScustomWorks** [comment]
I understand, here are the details and thanks for your hard work on this modules ;):

panel background: rgb 205, 207, 202
red texts: rgb 168, 15, 15

font: Ubuntu, https://www.fontsquirrel.com/fonts/ubuntu

Module name : Ubuntu bold 16pt
All caps labels: Ubuntu bold 12pt
Regular labels: Ubuntu bold 9pt

**Comment by Aepelzen** [comment]
I tried your panel design (except for the red colors) in the latest QuadSequencer update and so far I like it. Could you send me an SVG of your GateSequencer panel so I don't have to redo the whole thing?

Also, you seem to be pretty good at graphics stuff (at least better than me). If you have any suggestions to make the new QuadSequencer more clearly laid  out (without a drastic increase in size) I would love to hear them.

Thanks.

**Comment by AScustomWorks** [comment]
Yes, not convinced about the red too, went for it to make the module stand up against any other design, maybe blue?
here are the gate and quad panes as I made them before your last update, I'll see about Quad new version later ;)
[aep-panels.zip](https://github.com/Aepelzen/AepelzensModules/files/1876614/aep-panels.zip)

**Comment by AScustomWorks** [comment]
Oh, just took a look at the new Quad... I have a couple ideas that would make the labeling cleaner but keep the module the same size, I'll try and post it here asap

**Comment by Aepelzen** [comment]
Awesome. Thanks again.

I don't think I need any color for the fonts. I like a simple look so unless you have an awesome idea i will probably stick with that ^^

For the Quad I was thinking about having all the labels in one column somewhere on the side (like one label per row of knobs) but so far I could not find a knob arrangement where that would fit. And btw. there is no need to hurry, I have some other things to keep me busy while waiting for your suggestions.

**Comment by AScustomWorks** [comment]
Tried the one label only idea too but didn't like the results so, how bout this?
I made this changes:
 7pt for Range, Prob and Mode texts
And to give it a more minimal look:
Text labels rgb 50, 50, 50 (Module name still black)
lines rgb 177, 177,  177

![quadseq](https://user-images.githubusercontent.com/34798768/38406442-8f6dc2f6-393a-11e8-839a-9345e9ab1e30.jpg)

[QuadSeq.svg.zip](https://github.com/Aepelzen/AepelzensModules/files/1882942/QuadSeq.svg.zip)

**Comment by Aepelzen** [comment]
Yeah. This looks nice. I will probably keep the horizontal lines black (or at least the new text color) to have a clearer separation of the different "blocks" but other than that I like it alot.

I'm  feeling a bit under the weather today but I will try this asap. Thanks once again. I owe you one. If you ever need help with your modules (maybe in the programming/testing area instead of design), don't hesitate to ask.

**Comment by AScustomWorks** [comment]
Hi there, I'm liking your sampler module a lot, it would be nice to have you back on the plugin manager ;)
And, I went at it again(with the rest of the panels), if you want the files just let me know:
![ap-modules](https://user-images.githubusercontent.com/34798768/39025126-d7c94ca2-440a-11e8-9d9d-235593f02154.jpg)

**Comment by Aepelzen** [comment]
Hi, that's good to hear but I won't join the plugin manager. This was an interesting challenge, like one of those spot the difference riddles. So far I haven't found too many but it's still early in the  morning, I might try again later ^^

I like what you did with Erwin though. I haven't updated that panel yet because I'm still looking for a place to squeeze one more button in there. A small LED-Button would be enough. Any ideas on that? So far the only place I could think of is the spot where you put that Erwin label but I think it would be confusing to have a totally unrelated button within the scale. The only other free spot seems to be between the scale-select knob and input but that looked weird.

**Comment by AScustomWorks** [comment]
Hehe, yes, Erwin is a tough panel! Made a quick photoshop mockup rearranging octave and semi components positions to show a couple of possible spots for that switch.
The spot on the top wouldn't require to flip the keyboard, but flipping it makes the bottom spot available too, I hope this helps you ;)

![erwin](https://user-images.githubusercontent.com/34798768/39057671-70d14a66-447f-11e8-81a6-aab7ebd709f4.jpg)

**Comment by Aepelzen** [comment]
It does (as usual ^^). I actually tried the top position myself but for some reason yours somehow looks better than mine (also as usual I guess). I'm currently busy with another new module but I will get back to this.
